Sources/Orbit/Support/Components/TimelineStep.swift

```diff
@@ -147,32 +185,59 @@ public extension TimelineStep where Header == TimelineStepBadgeText, Content ==
 // MARK: - Types
 
 public enum TimelineStepStyle: Equatable {
-    case `default`
-    case status(Status)
+    case inactive
```

Comment:
State/variant names should be taken 1:1 from design when possible, to avoid creating arbitrary names.

Reply:
The enum will not prevent caller to use steps in wrong order or impossible combinations. We can keep the "status" property for each step (based on design - "default", "active", "success",...) and the information about which step is active can either be done similar to how the Tags work (preferred) - with an `index` parameter of the whole Timeline (this makes sure only one step is "current" and makes it easier to map this information from client), or it could be a second parameter (isActive) of each step.

Reply:
Changed cases to `past, current(Status), future`